src/stores/WidgetStore.ts

```diff
@@ -118,6 +118,7 @@ export default class WidgetStore extends AsyncStoreWithClient<IState> {
     }
 
     private loadRoomWidgets(room: Room) {
+        if (!room) return;
```

Comment:
would be good to link to https://github.com/vector-im/element-web/issues/8933 as the user is a guest when peeking

Reply:
I don't think this null-guard has anything to do with that however, given that peeking guests have valid rooms too.
I saw a `getRooms()` call return `[null]` which is what I am guarding against here.

Reply:
uh, that sounds like a larger problem that shouldn't be resolved in the depths of the widget code - that'll break the room list and pretty much everything else too.

Reply:
Indeed, I hit it once and could not repro the next 10x times. This is the only error log that came of it however.